### PR TITLE
Update the public subnet with a variable cidr and allocation pool.

### PR DESCRIPTION
**Pull request by mbruzek**
@jkilpatr please take a look

### CODE REVIEW

**Comment by jkilpatr** [comment]
that works but why not just make it a var? I guess it doesn't really matter.

**Comment by jeremyeder** [comment]
It does matter a little because even /22 is not going to be big enough eventually.

**Comment by jkilpatr** [comment]
so we need to do a stack update as well?

**Comment by mbruzek** [comment]
I changed the cidr to match the templates (/16) here: https://github.com/redhat-performance/openstack-templates/pull/14

**Comment by jkilpatr** [comment]
ok then, I think that should work.

**Comment by mbruzek** [comment]
Thanks @jtaleric I suspected that so I tested a similar range on openstack and it would not allow me to set 255 so I changed it in the PR already.

**Comment by smalleni** [comment]
@mbruzek Looks good now, only thing you've got to remember is our CIDR for ExternlpSubnet in netowrk-environment.yaml here ( https://github.com/jkilpatr/tripleo-quickstart-scalelab/blob/43c81ea8948ae7ff98ab2004e939517e9d887e48/profiles/main.yml#L52) should match with CIDR here (https://github.com/redhat-performance/openstack-templates/pull/14/files/#l40). Let me know if you need more details.


Also for the 172.21.0.0/16 subnet 1721.21.255.255 becomes the broadcast address. The subnet identifier 1721.21.0.0 and the broadcast address 172.21.255.255 cannot be used in the openstack virtual networks just like in physical networks.

**Comment by smalleni** [comment]
@mbruzek Can you squash all these commits please?

**Comment by jtaleric** [comment]
@mbruzek Yup, I saw the change. I just wanted to provided you with the reason why you could not use .255.

File: roles/undercloud-install/templates/introspect.sh.j2 [hunk 1]
```diff
@@ -107,6 +107,18 @@ clean()
                                 on_count=$(ironic --json node-list|jq -r ".[]| select(.[\"provision_state\"] == \"clean wait\")| .[\"uuid\"]" | wc -l) ; \
                             done'
     set -e
+
+    for node in `ironic --json node-list | jq -r '.[]| select(.["provision_state"] == "clean failed")|.["uuid"]'`; do
+        ironic node-set-provision-state $node manage
+        sleep 10
+    done
+    for node in `ironic --json node-list | jq -r '.[]| select(.["provision_state"] == "clean wait")|.["uuid"]'`; do
+        ironic node-set-maintenance $node true
```
Comment (smalleni) [inline roles/undercloud-install/templates/introspect.sh.j2]: Can we use openstack baremetal api please?

**Comment by mbruzek** [comment]
Do not merge. I screwed up the squash.

File: roles/undercloud-install/templates/introspect.sh.j2 (hunk 1 above)
Comment (mbruzek) [reply roles/undercloud-install/templates/introspect.sh.j2]: This isn't part of my change, it is a squash error

**Pull request closed without merging**